frontend/src/components/Header.js

```diff
@@ -11,7 +13,21 @@ const Header = () => {
           : "header vendor-header"
       }
     >
-      {location.pathname === "/" ? "Piacok" : "Árusok"}
+      {location.pathname === "/" ? (
```

Comment:
Ne már, erre van a Route komponens, itt a Headeren belül is lehet Routes-ot használni. Egyébként attól, hogy nem "/" a path még nem tuti, hogy az árusok oldalon vagyunk.